### ISSUE TITLE
Working with Multiple database not possible

### ISSUE THREAD

**Issue by rocky-bgta** [RobinBuschmann/sequelize-typescript#229]
This ORM work fine for single database. But when I want to change database; By changing **db** connection string it not work at all. ORM still reference to first database connection string. 
For example:

```
  dbConfig = {
    host: "localhost",
    port: 5432,
    name: 'rocky',
    dialect: 'postgres',
    username: 'postgres',
    password: '123456',
    modelPaths: [
        __dirname + '/entities/startUp',
        __dirname + '/entities/accounting'],
    logging:  true, //false
    pool: {
        max: 10,
        min: 1,
        idle: 10000
    },
    define: {
        freezeTableName: true,
        timestamps: true
    }
});

```
let  db =new Sequelize(dbConfig);
await db.sync();

above code work fine. 
But if I change **dbConfig** object and change database name property and try to execute following code
 
```
let newDbConfig = _.extend(dbConfig,dbName);

db =new Sequelize(newDbConfig);
await db.sync();
```

code execute fine but still work first database, that is 'rocky'

So please let me know is it possible to change **_db connection string_** through this ORM?

**Comment by RobinBuschmann** [comment]
Hi @rocky-bgta, do you use one sequelize instance per db connection?

It should be possible to connect to different DB’s using different sequelize instances and - which is really important - different models. So one model does only work for on sequelize instance (https://github.com/RobinBuschmann/sequelize-typescript/blob/master/README.md#recommendations-and-limitations)

**Comment by rocky-bgta** [comment]
Yes I want to use one sequlize instance for different db connection and I
want use the models. Is this possible ?

Thanks



Thanks
Software Engineer
Md. Nazmus Salahin Rocky

**Comment by RobinBuschmann** [comment]
@rocky-bgta You cannot use the same models for different sequelize instances currently.

**Comment by rocky-bgta** [comment]
Thanks for your kind information.



Thanks
Software Engineer
Md. Nazmus Salahin Rocky

**Comment by RobinBuschmann** [comment]
We're currently discussing an implementation to make same models for different sequelize instances possible. see #231

**Comment by rhogeranacleto** [comment]
:pensive:

**Comment by nicgirault** [comment]
Any update about this feature? My use case is that I've some scripts syncing some data from production to staging environment so I need 2 sequelize connections with the same model. Right now I'm rewriting these scripts to bypass models by writing raw queries. It's quite frustrating missing all the model features.

**Comment by RobinBuschmann** [comment]
Hey @nicgirault, I've published a new alpha of the upcoming major version which includes the repository mode: `sequelize-typescript@1.0.0-alpha.7` or `sequelize-typescript@canary`. I'm happy for any feedback. See documentation [here](https://github.com/RobinBuschmann/sequelize-typescript/tree/1.0.0#repository-mode)

**Comment by RobinBuschmann** [comment]
Since this won't be implemented in `0.6.X` this issue is closed
